### ISSUE TITLE
object contains unparsed element in datadog_integration_aws_tag_filter resource

### ISSUE THREAD

**Issue by tsub** [DataDog/terraform-provider-datadog#2039]
### Datadog Terraform Provider Version

v3.28.0

### Terraform Version

v1.5.4

### What resources or data sources are affected?

* datadog_integration_aws_tag_filter

### Terraform Configuration Files

```hcl
resource "datadog_integration_aws_tag_filter" "lambda" {
  account_id     = local.aws_account_id
  namespace      = "lambda"
  tag_filter_str = "datadog:monitored"
}
```

### Relevant debug or panic output

```
╷
│ Error: object contains unparsed element: map[namespace:elb tag_filter_str:[]]
│ 
│   with datadog_integration_aws_tag_filter.lambda,
│   on datadog-integration-aws.tf line 10, in resource "datadog_integration_aws_tag_filter" "lambda":
│   10: resource "datadog_integration_aws_tag_filter" "lambda" {
│ 
╵
```

### Expected Behavior

plan/apply succeeds without error

### Actual Behavior

plan/apply terminates with an error.

### Steps to Reproduce

I don't think it was an error until last week. The error started showing up this week. 🤔 

1. `terraform plan` or `terraform apply`

### Important Factoids

The following tag filters have already been configured in Datadog's AWS integration configuration screen.

![image](https://github.com/DataDog/terraform-provider-datadog/assets/10208211/b0cf9338-0fb3-4deb-a675-0269e2a4dd33)

**Comment by skarimo** [comment]
Hi, thanks for opening the issue. We have identified the issue and fix has been put in place. Im going to go ahead and close this issue but feel free to re-open if the issue persists. Thanks again!

**Comment by lloydmanansala** [comment]
### Datadog Terraform Provider Version
v3.46.0

### Terraform Version
v1.9.5

### What resources or data sources are affected?
```
resource "datadog_integration_aws_tag_filter" "additional_aws_filter_tag" {
  for_each       = <object>
  account_id     = var.account_id
  namespace      = each.value.namespace
  tag_filter_str = each.value.tags
}
```
### Relevant debug or panic output

I'm not sure if a fix has put in place as I'm seeing the issue. 
```╷
│ Error: object contains unparsed element: map[namespace:step_functions tag_filter_str:<nil>]
│ 
│   with datadog_integration_aws_tag_filter.additional_aws_filter_tag["0"],
│   on aws-dd-integration.tf line 28, in resource "datadog_integration_aws_tag_filter" "additional_aws_filter_tag":
│   28: resource "datadog_integration_aws_tag_filter" "additional_aws_filter_tag" {```

**Comment by mc-mmccants** [comment]
> I'm not sure if a fix has put in place as I'm seeing the issue.
> 
> ```
> │ Error: object contains unparsed element: map[namespace:step_functions tag_filter_str:<nil>]
> │ 
> │   with datadog_integration_aws_tag_filter.additional_aws_filter_tag["0"],
> │   on aws-dd-integration.tf line 28, in resource "datadog_integration_aws_tag_filter" "additional_aws_filter_tag":
> │   28: resource "datadog_integration_aws_tag_filter" "additional_aws_filter_tag" {```
> ```

Having the same exact issue with no changes to our configuration. We are using an older version of the provider, but updating did not help. I assume it's an issue with their API.

**Comment by JulesClaussen** [comment]
Seems like this started happening again (we noticed the issue a couple of minutes ago) @skarimo ?

**Comment by michelzanini** [comment]
It seems the issue has been fixed again in version `3.48.1`
